apps/spruce/src/pages/spawn/spawnHost/EditSpawnHostModal.tsx

```diff
@@ -103,13 +131,15 @@ export const EditSpawnHostModal: React.FC<EditSpawnHostModalProps> = ({
     hostId: host.id,
     myPublicKeys: publicKeys,
     oldUserTags: userTags,
+    timeZone,
   });
 
   const currEditState = formToGql({
     formData: formState,
     hostId: host.id,
     myPublicKeys: publicKeys,
     oldUserTags: userTags,
+    timeZone,
   });
```

Comment:
There's this weird bug where if you spawn an expirable host and then try to edit it, you can't click on the unexpirable button:

https://github.com/evergreen-ci/ui/assets/47064971/c4b72d51-7c14-4402-b08c-d09ec3688f56

Reply:
I'm also unable to make individual edits to this host e.g. if I just wanted to change the host name, I can't save it